unlock-app/src/components/interface/members/MemberCard.tsx

```diff
@@ -94,17 +94,26 @@ export const MemberCard: React.FC<MemberCardProps> = ({
     try {
       if (!storageService) return
       const { lockAddress, token: keyId } = metadata
-      const markTicketCheckInPromise = storageService.markTicketAsCheckedIn({
-        lockAddress,
-        keyId,
-        network: network!,
-      })
+      const checkIn = async () => {
+        const response = await storageService.markTicketAsCheckedIn({
+          lockAddress,
+          keyId,
+          network: network!,
+        })
+
+        if (!response.ok && response.status === 409) {
+          ToastHelper.error('Ticket already checked in')
+        }
+      }
+
+      const markTicketCheckInPromise = checkIn()
 
       await ToastHelper.promise(markTicketCheckInPromise, {
         loading: `Marking ticket as Check-in`,
         error: `Error on marking ticket as checked-in`,
         success: `Successfully marked ticket as checked-in`,
       })
+
       if (typeof loadMembers === 'function') {
         loadMembers()
       }
```

Comment:
```suggestion
     const response = await storageService.markTicketAsCheckedIn({
        lockAddress,
        keyId,
        network: network!,
      })

      if (!response.ok && response.status === 409) {
        ToastHelper.error('Ticket already checked in')
      }

      if (response.ok) {
        ToastHelper.success('Successfully marked ticket as checked-in')
        if (typeof loadMembers === 'function') {
          loadMembers()
        }
      }
```

Reply:
lets `loadMembers` only when ticket is marked as checked-in otherwise only show the error message